plugin/src/main/java/org/opensearch/sql/plugin/catalog/CatalogServiceImpl.java

```diff
@@ -34,6 +35,8 @@ public class CatalogServiceImpl implements CatalogService {
 
   private static final CatalogServiceImpl INSTANCE = new CatalogServiceImpl();
 
+  private static final String CATALOG_NAME_REGEX = "^[a-zA-Z0-9_-]*$";
```

Comment:
"" is invalid, right?

`assertTrue("".matches("^[a-zA-Z0-9_-]*$"));`

Reply:
Yes Invalid...there is an empty string check as well before the regex check. 

Will change the regex to "^[a-zA-Z0-9_-]+$"